File: app/src/main/java/com/example/cbanda/achieve/models/db/Quote.java
package com.example.cbanda.achieve.models.db;


import com.google.gson.annotations.Expose;
import com.google.gson.annotations.SerializedName;

public class Quote {

    @SerializedName("quoteText")
    @Expose
    private String quoteText;
    @SerializedName("quoteAuthor")
    @Expose
    private String quoteAuthor;
    @SerializedName("senderName")
    @Expose
    private String senderName;
    @SerializedName("senderLink")
    @Expose
    private String senderLink;
    @SerializedName("quoteLink")
    @Expose
    private String quoteLink;

    public String getQuoteText() {
        return quoteText;
    }

    public void setQuoteText(String quoteText) {
        this.quoteText = quoteText;
    }

    public String getQuoteAuthor() {
        return quoteAuthor;
    }

    public void setQuoteAuthor(String quoteAuthor) {
        this.quoteAuthor = quoteAuthor;
    }

    public String getSenderName() {
        return senderName;
    }

    public void setSenderName(String senderName) {
        this.senderName = senderName;
    }

    public String getSenderLink() {
        return senderLink;
    }

    public void setSenderLink(String senderLink) {
        this.senderLink = senderLink;
    }

    public String getQuoteLink() {
        return quoteLink;
    }

    public void setQuoteLink(String quoteLink) {
        this.quoteLink = quoteLink;
    }
}

File: app/src/main/java/com/example/cbanda/achieve/models/db/Quote.kt
package com.example.cbanda.achieve.models.db


import com.google.gson.annotations.Expose
import com.google.gson.annotations.SerializedName

class Quote {

    @SerializedName("quoteText")
    @Expose
    var quoteText: String? = null
    @SerializedName("quoteAuthor")
    @Expose
    var quoteAuthor: String? = null
    @SerializedName("senderName")
    @Expose
    var senderName: String? = null
    @SerializedName("senderLink")
    @Expose
    var senderLink: String? = null
    @SerializedName("quoteLink")
    @Expose
    var quoteLink: String? = null
}